modules/search/instant-search/lib/query-string.js

```diff
@@ -27,6 +27,51 @@ export function setSearchQuery( searchValue ) {
 	pushQueryString( encode( query ) );
 }
 
+export function getSearchSort() {
+	const query = getQuery();
+	const order = 'order' in query ? query.order : 'DESC';
+	const orderby = 'orderby' in query ? query.orderby : 'relevance';
+	let sort;
+	switch ( orderby ) {
+		case 'date':
+			if ( order === 'ASC' ) {
+				sort = 'date_asc';
+			} else {
+				sort = 'date_desc';
+			}
+			break;
+		case 'price':
+			if ( order === 'ASC' ) {
+				sort = 'price_asc';
+			} else {
+				sort = 'price_desc';
+			}
+			break;
+		case 'rating':
+			if ( order === 'ASC' ) {
+				sort = 'rating_asc';
+			} else {
+				sort = 'rating_desc';
+			}
+			break;
+		case 'recency':
+			sort = 'score_recency';
+			break;
+		case 'keyword':
```

Comment:
This `orderby` returns a 400; `message: "Invalid sort value: score_keyword"`.

Reply:
Hmmm, ya I guess i disabled that sort method because I wasn't sure if it made sense. Going to leave it in the code here for now. The idea behind it is to have a sorting that is only based on the term and document frequencies in the index.